src/jit/gentree.cpp

```diff
@@ -953,6 +953,27 @@ Compiler::fgWalkResult Compiler::fgWalkTreePostRec(GenTreePtr* pTree, fgWalkData
         }
         break;
 
+        case GT_LIST:
+        {
+            GenTreeArgList* list = tree->AsArgList();
+            if (list->IsAggregate())
+            {
+                for (; list != nullptr; list = list->Rest())
+                {
+                    result = fgWalkTreePostRec<computeStack>(&list->gtOp1, fgWalkData);
+                    if (result == WALK_ABORT)
+                    {
+                        return result;
+                    }
+                }
+                break;
+            }
+            else
+            {
+                __fallthrough;
+            }
+        }
```

Comment:
The
```
            else
            {
                __fallthrough;
            }
```
construct seems weird. I would simply use:

```
            __fallthrough;
```

with surrounding blank lines.